starfish/intensity_table.py

```diff
@@ -57,9 +56,22 @@ class IntensityTable(xr.DataArray):
     # constant that stores the attr key for the shape of ImageStack
     IMAGE_SHAPE = 'image_shape'
 
+    @staticmethod
+    def _build_coordinates(
```

Comment:
i got confused here, coordinates here doesn't mean Coordinates.X, Y, Z, etc. But rather, it means Xarray Coordinates right? If so, consider re-naming to _build_xarray_coordinates?

Reply:
That's correct. Will fix. 👍

Reply:
actually, idiomatic xarray calls coordinates, coords. so maybe _build_xarray_coords